tests/test-sources/plugins/languages/markdown/glow.nix

```diff
@@ -0,0 +1,26 @@
+{ pkgs, helpers, ... }:
+{
+  empty = {
+    plugins.glow.enable = true;
+  };
+
+  defaults = {
+    plugins.glow = {
+      enable = true;
+
+      glowPackage = pkgs.glow;
+
+      settings = {
+        glow_path = helpers.mkRaw "vim.fn.exepath('glow')";
```

Comment:
```suggestion
        glow_path.__raw = "vim.fn.exepath('glow')";
```

Reply:
are these not equivalent? or is helpers.mkRaw not preferred?

Reply:
They are equivalent, so it's a style nit.

We usually use `.__raw =` when it can be set as a nested attr like this.

Reply:
(Yeah I'd say it's a nit because I would personally prefer the helpers.mkRaw, as I find it less confusing to read)